src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int colorIndex = new Random().nextInt(DifferentColors.values().length);
```

Comment:
The 'DifferentColors.values().length' should be stored as a constant to avoid calling 'values()' multiple times which creates a new array each time.